example.config.json

```diff
@@ -1,14 +1,20 @@
 {
-"daiAddress": "0x6b175474e89094c44da98b954eedeac495271d0f",
-"usdcAddress": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
-"usdtAddress": "0xdac17f958d2ee523a2206206994597c13d831ec7",
-"wethAddress": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-"raiAddress": "0x03ab458634910aad20ef5f1c8ee96f1d6ac54919",
-"opolisHelper": "",
-"opolisAdmin": "",
-"wyreAddress": "",
-"privateKey": "",
-"infura": "",
-"etherscan": "",
-"coinmarketcap": ""
-}
\ No newline at end of file
+  "daiAddress": "0x6b175474e89094c44da98b954eedeac495271d0f",
+  "usdcAddress": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+  "usdtAddress": "0xdac17f958d2ee523a2206206994597c13d831ec7",
+  "wethAddress": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+  "raiAddress": "0x03ab458634910aad20ef5f1c8ee96f1d6ac54919",
+  "opolisHelper": "",
+  "opolisAdmin": "",
+  "wyreAddress": "",
```

Comment:
We can probably get rid of Wyre address as long as all of the liquidation addresses can be the same.